10up/simple-local-avatars
# Add a test for the front end

### Description of the Change
PR adds E2E tests for checking the front end of avatars

Closes #188 

### How to test the Change

1. Check out the branch
2. Run `npm install` and `npm run build`
3. Start the WP environment with `npm run env:start`
4. Run the tests with `npm run cypress:run`
5. Verify all tests pass

### Changelog Entry

> Added - E2E test for checking the front end of avatars

### Credits

Props @Firestorm980 


### Checklist:
